code/custom-vgg16-gpu.ipynb

```diff
@@ -143,54 +151,54 @@
-      "conv2d_21 (Conv2D)           (None, 13, 13, 512)       1180160   \n",
+      "conv2d_8 (Conv2D)            (None, 3, 13, 512)        590336    \n",
       "_________________________________________________________________\n",
-      "conv2d_22 (Conv2D)           (None, 13, 13, 512)       2359808   \n",
+      "conv2d_9 (Conv2D)            (None, 3, 13, 512)        2359808   \n",
       "_________________________________________________________________\n",
-      "conv2d_23 (Conv2D)           (None, 13, 13, 512)       2359808   \n",
+      "conv2d_10 (Conv2D)           (None, 3, 13, 512)        2359808   \n",
       "_________________________________________________________________\n",
-      "max_pooling2d_9 (MaxPooling2 (None, 6, 6, 512)         0         \n",
+      "max_pooling2d_4 (MaxPooling2 (None, 3, 6, 256)         0         \n",
       "_________________________________________________________________\n",
-      "conv2d_24 (Conv2D)           (None, 6, 6, 512)         2359808   \n",
+      "conv2d_11 (Conv2D)           (None, 3, 6, 512)         1180160   \n",
       "_________________________________________________________________\n",
-      "conv2d_25 (Conv2D)           (None, 6, 6, 512)         2359808   \n",
+      "conv2d_12 (Conv2D)           (None, 3, 6, 512)         2359808   \n",
       "_________________________________________________________________\n",
-      "conv2d_26 (Conv2D)           (None, 6, 6, 512)         2359808   \n",
+      "conv2d_13 (Conv2D)           (None, 3, 6, 512)         2359808   \n",
       "_________________________________________________________________\n",
-      "max_pooling2d_10 (MaxPooling (None, 3, 3, 512)         0         \n",
+      "max_pooling2d_5 (MaxPooling2 (None, 3, 3, 256)         0         \n",
       "_________________________________________________________________\n",
-      "flatten_2 (Flatten)          (None, 4608)              0         \n",
+      "flatten_1 (Flatten)          (None, 2304)              0         \n",
       "_________________________________________________________________\n",
-      "dense_4 (Dense)              (None, 4096)              18878464  \n",
+      "dense_1 (Dense)              (None, 4096)              9441280   \n",
       "_________________________________________________________________\n",
-      "dense_5 (Dense)              (None, 4096)              16781312  \n",
+      "dense_2 (Dense)              (None, 4096)              16781312  \n",
       "_________________________________________________________________\n",
-      "dense_6 (Dense)              (None, 37)                151589    \n",
+      "dense_3 (Dense)              (None, 37)                151589    \n",
       "=================================================================\n",
-      "Total params: 50,526,053\n",
-      "Trainable params: 50,526,053\n",
+      "Total params: 39,194,405\n",
+      "Trainable params: 39,194,405\n",
```

Comment:
By reorganizing the input channels we require significantly fewer network parameters.